tests/py/requirements.txt

```diff
@@ -8,6 +8,6 @@ pytest>=8.2.1
 pytest-xdist>=3.6.1
 pyyaml
 timm>=1.0.3
-transformers==4.40.2
-nvidia-modelopt[deploy,hf,torch]~=0.17.0
+transformers==4.40.2; python_version < "3.13"
```

Comment:
Id rather upgrade this if we can just because its easy to forget about settings like this